compiler/p/codegen/PPCTableOfConstants.cpp

```diff
@@ -499,7 +499,7 @@ int32_t TR_PPCTableOfConstants::lookUp(TR::SymbolReference *symRef, TR::CodeGene
    int8_t           local_buffer[1024];
    int8_t          *name = local_buffer;
    bool             isAddr = false;
-   intptr_t        myTag;
+   intptr_t        myTag=0;
```

Comment:
Please add a space both sides of `=` (like the lines above).

Reply:
resolved